.eslintrc.json

```diff
@@ -1,4 +1,20 @@
 {
+  "root": true,
   "extends": "hexo",
```

Comment:
Would you please extends https://github.com/hexojs/eslint-config-hexo/blob/de333162ba947a9bf819f98bb9394b0b49041e26/ts.js ?

```diff
- "extends": "hexo",
+ "extends": "hexo/ts.js",  
```